Source/Redemption/Public/RedemptionPlayer.h

```diff
@@ -5,6 +5,7 @@
 #include "CoreMinimal.h"
 #include "GameFramework/Character.h"
 #include "Components/WidgetComponent.h"
+#include "Engine/GameInstance.h"
```

Comment:
please include this in the cpp file there is no reason for it to be in the h file